onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/VehicleOverlay.java

```diff
@@ -91,25 +92,30 @@
 
     private static final int NORTH = 0;
 
-    private static final int NORTH_WEST = 1;
+    private static final int NORTH_EAST = 1;
 
-    private static final int WEST = 2;
+    private static final int EAST = 2;
 
-    private static final int SOUTH_WEST = 3;
+    private static final int SOUTH_EAST = 3;
 
     private static final int SOUTH = 4;
 
-    private static final int SOUTH_EAST = 5;
+    private static final int SOUTH_WEST = 5;
 
-    private static final int EAST = 6;
+    private static final int WEST = 6;
 
-    private static final int NORTH_EAST = 7;
+    private static final int NORTH_WEST = 7;
 
     private static final int NO_DIRECTION = 8;
 
     private static final int NUM_DIRECTIONS = 9; // 8 directions + undirected mVehicles
 
-    private static Bitmap[] vehicle_icons = new Bitmap[NUM_DIRECTIONS];  // Black vehicle icons
+    private static final int DEFAULT_VEHICLE_TYPE = ObaRoute.TYPE_BUS; // fall back on bus
+
+    // Vehicle type (if available) -> icon set
+    private static Map<Integer, Bitmap[]> vehicle_type_icons = new HashMap<>(ObaRoute.NUM_TYPES);
+
+//    private static Bitmap[] vehicle_icons = new Bitmap[NUM_DIRECTIONS];  // Black vehicle icons
```

Comment:
Looks like this is left over from the previous implementation, you can just remove it.